pkgs/by-name/ch/chasemapper/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  python3,
+}:
+let
+  customPython = python3.withPackages (
+    ps: with ps; [
+      eccodes
+      cusfpredict
+      flask
+      flask-socketio
+      lxml
+      numpy
+      python-dateutil
+      pytz
+      requests
+      pyserial
+      simple-websocket
+    ]
+  );
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "chasemapper";
+  version = "1.5.3";
+
+  src = fetchFromGitHub {
+    owner = "projecthorus";
+    repo = "chasemapper";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-KamXFMwozCQnH4zQRqsxq28Jr1NEjDL/Jimy/XEP0Bs=";
+  };
+
+  postPatch = ''
+    substituteInPlace horusmapper.py \
+      --replace-fail '#!/usr/bin/env python2.7' '#!${customPython}/bin/python3'
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp horusmapper.py $out/
+    cp -r chasemapper $out/
+    cp -r static $out/
+    cp -r templates $out/
+    chmod +x $out/horusmapper.py
+    echo $out/horusmapper.py '"$@"' > $out/bin/chasemapper
+    chmod +x $out/bin/chasemapper
```

Comment:
You should replace some `cp` and `chmod` with a simple `install`.

Reply:
Updated

Reply:
Please mark all the feedback of this PR as resolved when you think they are resolved.

Reply:
I normally leave things unresolved until the original person can re-review and verify that it's fixed to their specification. But I will keep in mind in the future that nixpkgs etiquette is different!